Docker/OpenWRT/runTests.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/sh
+
+function runCMD() { # usage: runCMD "<command>" "<retVal>"
+    eval $1 &>/dev/null
+    RETVAL=$?
+    if [ "$RETVAL" != "$2" ]; then
+        echo "Command ($1) returned ${RETVAL}, but expected $2. Rerunning with output to terminal:"
+        eval $1
+        exit 1
+    fi
+}
+
+# Successful tests
+runCMD "ldd /lib/libustream-ssl.so" 0
+runCMD "opkg update" 0
+runCMD "uclient-fetch -O /dev/null 'https://letsencrypt.org'" 0
+# Negative tests
+runCMD "uclient-fetch --ca-certificate=/dev/null -O /dev/null 'https://letsencrypt.org'" 5
+runCMD "uclient-fetch -O /dev/null 'https://self-signed.badssl.com/'" 5
+runCMD "uclient-fetch -O /dev/null 'https://untrusted-root.badssl.com/'" 5
+runCMD "uclient-fetch -O /dev/null 'https://expired.badssl.com/'" 5
```

Comment:
Just FYI, I can create a PR myself later to improve the coverage. In the email conversation I've provided two more examples:

```
$ opkg install auc && auc -c
$ opkg install luci-ssl && uclient-fetch --no-check-certificate https://localhost/
```

`auc` is just another client, so similar to `uclient-fetch`.

The `luci-ssl` one is probably more important, because it's testing TLS on `uhttpd` web server, so testing server side API/ABI.

Reply:
I didn't include these tests because when running the `auc` command, it tried to access `ubus` which gave an error. The `luci-ssl` webserver doesn't run during a build. Running the container in a GitHub Action gave some errors for me as well since it doesn't have the expected entrypoint that the GitHub wrappers expect.

Reply:
> I didn't include these tests because when running the auc command, it tried to access ubus which gave an error. 

Indeed, interesting, I wasn't aware about those issues.